Upgrade electron-notarize and vars

rly opened the pull request:
Fix #527

- [x] Finish testing locally
- [x] ~Merge PR and~ test "Build-and-deploy-mac" GitHub Action

CodyCBakerPhD commented:
> Merge PR and test "Build-and-deploy-mac" GitHub Action

You don't need to merge to test this

CodyCBakerPhD commented:
Building now, check back in an hour. Will show up under v0.0.13 draft release

rly commented:
Ah I forgot you could do that! Nice, it worked. I had previously set the version to 0.0.14 because that was the latest draft. It looks like draft 0.0.13 was deleted, so I've changed the version here to 0.0.13 and we can delete 0.0.14. I also added a description.
